docs/server-startup.md

```diff
@@ -1,24 +1,24 @@
 # Starting TabPy Server
 
-These instructions explain how to start up TabPy Server.
-
-<!-- toc -->
-
-- [Prerequisites](#prerequisites)
-- [Windows](#windows)
-  * [Command Line Arguments](#command-line-arguments)
-- [Mac](#mac)
-  * [Command Line Arguments](#command-line-arguments-1)
-- [Linux](#linux)
-  * [Command Line Arguments](#command-line-arguments-2)
-
-<!-- tocstop -->
-
+These instructions explain how to start up TabPy Server.
+
+<!-- toc -->
+
+- [Prerequisites](#prerequisites)
+- [Windows](#windows)
+  * [Command Line Arguments](#command-line-arguments)
+- [Mac](#mac)
+  * [Command Line Arguments](#command-line-arguments-1)
+- [Linux](#linux)
+  * [Command Line Arguments](#command-line-arguments-2)
+
+<!-- tocstop -->
+
 ## Prerequisites
 
 To start up TabPy Server from an environment the following prerequisites are required:
 
-- Python 3 or greater
+- Python 3.7.2

```

Comment:
Just out of curiosity what was the decision to pin specifically to this version? I'm not aware of any other projects that make such a tight restriction down to a minor version, and most enterprise customers I would think may not be able to update to such a new version.

FWIW if deterministic builds is the goal here then I would suggest taking another look at Pipenv. It was recently removed with the major updates but would help with what I think you are trying to achieve here

Reply:
Hi Will! Good question. Currently we are listing Python 3.7.2 as a prerequisite because that is the version we are testing internally. We are currently in the process of determining a range of versions that we can commit to supporting.

We'll look into Pipenv, thanks for the suggestion!